apps/system/js/modal_dialog.js

```diff
@@ -362,23 +362,24 @@ var ModalDialog = {
 
   buildSelectOneDialog: function md_buildSelectOneDialog(data) {
     var elements = this.elements;
-    elements.selectOneTitle.textContent = data.title;
-    elements.selectOneMenu.innerHTML = '';
+    this._localizeElement(elements.selectOneTitle, data.title);
```

Comment:
I don't understand why all other modals use `evt.detail.title` but "selectone" uses `evt.detail.message.title`. If there isn't a good reason, could you please fix it? Also might be worth to make `setTitle` use `_localizeElement` and use it here (or use it in the `case 'selectone'` line 239 to do like the other modals).